sample/kotlin/src/main/res/values/arrays.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+  ~ This product includes software developed at Datadog (https://www.datadoghq.com/).
+  ~ Copyright 2016-Present Datadog, Inc.
+  -->
+
+<resources>
+    <string-array name="genders">
+        <item>Female</item>
+        <item>Male</item>
+    </string-array>
```

Comment:
We want to be inclusive and open to gender diversity. Event if it's a sample app, let's make the choice a free form Text field to let people use the exact gender they want. 
I know we don't have any user for this app but it still provides a sample that will show a more inclusive way of thinking.